src/components/EstimateOption.tsx

```diff
@@ -0,0 +1,63 @@
+import { Text, nullable } from '@taskany/bricks';
+import { gapXs, gapS, gray9, gray7 } from '@taskany/colors';
+import { IconPlusCircleSolid } from '@taskany/icons';
+import React, { ReactNode, useCallback } from 'react';
+import styled from 'styled-components';
+
+const StyledWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${gapXs};
+`;
+
+const StyledContent = styled.div`
+    display: flex;
+    align-items: end;
+    gap: ${gapS};
+`;
+
+const StyledIconPlusCircleSolid = styled(IconPlusCircleSolid)`
+    cursor: pointer;
```

Comment:
This looks like a bug. Icon with onClick must have this style. Please check it, create issue in bricks if not and place comment with issue link here.